documentation/src/main/markdown/packagemgrs/maven.md

```diff
@@ -12,14 +12,19 @@
 * Maven Wrapper CLI
 * Maven Project Inspector
 
+<note type="Note">
+
+* Maven Project Inspector relies on Project Inspector thus does not accept Maven specific configuration properties.
+</note>
+
 ## Maven CLI
 
 The Maven CLI detector discovers dependencies of Maven projects by executing mvn commands.
 
-The Maven CLI detector attempts to run on your project if it finds a pom.xml file in the source directory (top level). It requires either mvnw or mvn:
+The Maven CLI detector attempts to run on your project if it finds a pom.xml file in the top level source directory and requires either `mvnw` or `mvn`.
```

Comment:
Just 'Attempts...'?

Reply:
Technically, as it may fail, therefore only attempting initially. ; )
Will tweak this.